tests/vxlan/test_vnet_vxlan.py

```diff
@@ -50,7 +50,7 @@ def prepare_ptf(ptfhost, mg_facts, dut_facts, vnet_config):
         "minigraph_vlan_interfaces": mg_facts["minigraph_vlan_interfaces"],
         "dut_mac": dut_facts["ansible_Ethernet0"]["macaddress"],
         "vnet_interfaces": vnet_config["vnet_intf_list"],
-        "vnet_routes": vnet_config["vnet_route_list"],
+        "vnet_routes": vnet_config["vnet_route_list"]+vnet_config["vnet_subnet_routes"],
```

Comment:
Add spaces for consistency with rest of file
```suggestion
        "vnet_routes": vnet_config["vnet_route_list"] + vnet_config["vnet_subnet_routes"],
```